pyspecProcScripts/third_level/process_enhancement.py

```diff
@@ -134,6 +126,13 @@ def process_enhancement(s, searchstr='', signal_pathway = {'ph1':1},
     d = d['t2':(0,t_range[-1])]
     d['t2':0] *= 0.5
     d.ft('t2')
+    if fl is not None:
+        fl.next('FID sliced')
+        fl.image(d.C.setaxis(
+'power','#').set_units('power','scan #'))
+    d *= sign
```

Comment:
you can't call this variable `sign` since `sign` is a numpy function -- it will overwrite and give weird results

Reply:
this makes sooooo much sense!! thank you!